internal/hud/hud.go

```diff
@@ -37,6 +38,8 @@ type Hud struct {
 	currentViewState view.ViewState
 	mu               sync.RWMutex
 	isRunning        bool
+
+	currentProfileFile *os.File
```

Comment:
is this used anywhere?

Reply:
cruft from an earlier iteration. thanks!